src/Core/Services/TypeHelper.cs

```diff
@@ -227,7 +227,9 @@ public static JsonDataType GetJsonDataTypeFromSystemType(Type type)
         /// <exception>Failed type conversion.</exception>"
         public static Type GetSystemTypeFromSqlDbType(string sqlDbTypeName)
         {
-            if (Enum.TryParse(sqlDbTypeName, ignoreCase: true, out SqlDbType sqlDbType))
+            // Remove the length specifier from the type name if it exists.Example: varchar(50) -> varchar
+            string baseType = sqlDbTypeName.Split('(')[0];
+            if (Enum.TryParse(baseType, ignoreCase: true, out SqlDbType sqlDbType))
```

Comment:
Consider the performance remarks about `string.Split(...)` because this is code is not restricted to just DW. What are your thoughts on the suggested:

> To split a string at a separator string, use the IndexOf or IndexOfAny method to locate the first character of the separator string. Then use the Compare method to determine whether the characters after that first character are equal to the remaining characters of the separator string.

https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-6.0#system-string-split(system-char()):~:text=To%20split%20a%20string%20at%20a%20separator%20string%2C%20use%20the%20IndexOf%20or%20IndexOfAny%20method%20to%20locate%20the%20first%20character%20of%20the%20separator%20string.%20Then%20use%20the%20Compare%20method%20to%20determine%20whether%20the%20characters%20after%20that%20first%20character%20are%20equal%20to%20the%20remaining%20characters%20of%20the%20separator%20string.

Reply:
Thanks for this suggestion. fixed!

Reply:
Thanks for this suggestion. added the fix.